app/helpers/course_summaries_helper.rb

```diff
@@ -32,12 +46,10 @@ def get_table_json_data
         assignment_marks: marks,
         grade_entry_form_marks: gef_marks,
         weighted_marks:
-          get_weighted_total_for_all_marking_schemes_for_student(
-            all_schemes, marks, gef_marks, weights, gef_weights, max_marks,
-            gef_max_marks)
-      }
-    end
-    student_list.to_json
+            get_weighted_total_for_all_marking_schemes_for_student(
+                @all_schemes, marks, gef_marks, @weights, @gef_weights, @max_marks,
```

Comment:
Line is too long. [83/80]<br>Indent the first parameter one step more than the previous line.